core/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala

```diff
@@ -408,6 +408,15 @@ trait DeltaSQLConfBase {
       .booleanConf
       .createWithDefault(true)
 
+  val DELTA_CHECKPOINT_MAX_ACTIONS_PER_FILE =
+    buildConf("checkpoint.maxActionsPerFile")
```

Comment:
nit: since we call this `part`s in our code (such as checkpointFileWithParts/numCheckpointParts), I prefer to call this `checkpoint.partSize`.

Reply:
changed the config name